languages/c/reference/exercise-concepts/two-fer.md

```diff
@@ -0,0 +1,22 @@
+# Concepts of two-fer
+
+```c
+#include "two_fer.h"
+#include <stdio.h>
+
+void two_fer(char *buffer, const char *name) {
+    sprintf(buffer, "One for %s, one for me.", name ? name : "you");
+}
+```
```

Comment:
In terms of the format of the concept document, it's handy to see the code here for this short implementation, but this is not sustainable for a uniform approach across all exercises. I see most of the other tracks use a link to the exercise implementation file for this, perhaps this would be a more sustainable approach? See this C# PR for example: https://github.com/exercism/v3/pull/357

Reply:
Yes, that would be better!